### ISSUE TITLE
Loading mask, rendering jumpiness

### ISSUE THREAD

**Issue by runfaj** [amcharts/amcharts4#1605]
I've got quite a few charts on a dashboard (mix of v3 and v4) and they are rendering well enough. However, when you get a data series over maybe 500 points, the rendering takes awhile to actually finish. This is fine, but I'm finding that there isn't a way to hide the resulting jumpiness in rendering from the user.

Take this example: https://recordit.co/q43RK3wjIp I implemented the loading indicator as outlined here: https://www.amcharts.com/docs/v4/tutorials/custom-loading-indicator/. Once the chart ready event fires, I dispose of the loading indicator.

The loading message appears and is present immediately, but as you can see in this recording, sometimes the loading background fill doesn't appear at all. Sometimes it appears as a flicker before the final render , but never does it appear before you can see grid lines, axis labels, etc. Here's another example after clearing cache that makes the rendering jumpiness even more noticable: https://recordit.co/61SAIVAvF5

Roughly, in my chart rendering, here's the order of what I'm setting up, similar to most of the examples in tutorials in the docs:
1. initialize xychart with data
1. show loading indicator
2. set chart colors
3. load category x axis
4. load 1 or 2 value y axes
5. load 1 or more series
6. load cursor
7. load legend

So my question(s) are:
- Can amcharts prerender a chart in the background and only show it when all the calculations are done?
- If not, and the loading mask is the way to go, is there a way to force that to fully render first to allow the fill to appear?

If neither of those are options, I can always place an extra div in my parent container that sits over the top and waits for the ready event to fire on the chart - but wanted to see if there was a cleaner option.

**Comment by martynasma** [comment]
Have you tried queuing chart renders?

https://www.amcharts.com/docs/v4/concepts/performance/#Daisy_chaining_multiple_charts

**Comment by runfaj** [comment]
@martynasma The queue option doesn't fix the fill issue. I was curious to see if it would improve loading times, so I took some timings of my dashboard that contains 3 charts from v3 and their equivalent in v4. I initialized the 6 charts roughly the same time via controlled react components, so I can accurately time the loading of each one. For my timing, I put a duration timer that starts when the loading indicator is added, then logs out once the ready event fires on each chart. I then refreshed 4 times so I could get a relative baseline on timing.

Here's with the queued flag off:
![image](https://user-images.githubusercontent.com/3292053/63947694-a4974d00-ca34-11e9-8bc0-379aa602e382.png)

Here's with the queued flag on:
![image](https://user-images.githubusercontent.com/3292053/63947495-40748900-ca34-11e9-8347-87b194bfa320.png)

So timing is pretty consistent regardless, but the jumpiness is still a visible issue.
